inc/sprite_eng_legacy.h

```diff
@@ -42,13 +42,20 @@
  *      instead of adding it in last position (default)
  */
 #define SPR_FLAG_INSERT_HEAD                    0x4000
+/**
+ *  \brief
+ *      Disable animation auto loop..<br>
+ *      By default animation always restart fater the last frame has been played.
+ *      This flag prevent the animation to restart and so the animation end on the last frame forever (see #SPR_getAnimationDone(..))
+ */
+#define SPR_FLAG_DISABLE_ANIMATION_LOOP         0x2000
 /**
  *  \brief
  *      Disable delaying of frame update when we are running out of DMA capacity.<br>
  *      By default we delay the frame update when DMA is already full so the frame update happen as soon we have enough DMA capacity to do it.
  *      This flag forces frame update to always happen immediately but that may cause graphical glitches.
  */
-#define SPR_FLAG_DISABLE_DELAYED_FRAME_UPDATE   0x2000
+#define SPR_FLAG_DISABLE_DELAYED_FRAME_UPDATE   0x8000
```

Comment:
Unfortunately you cannot use 0x8000 as flag marker as it's already used by the ALLOCATED state (L41 of sprite_eng_legacy.c) and it will conflict then. At worst you can use 0x0080 as update flags don't use it (NEED_UPDATE total mask = 0x001F) and you can still use 0x20 for ANIMATION_DONE marker.